Dockerfile

```diff
@@ -70,9 +77,8 @@ RUN Rscript -e "\
       paste0('$import_url', '$import_tag'), \
       paste0('$utils_url', '$utils_tag') \
     ); \
-  workflow_pkgs <- renv::dependencies('DESCRIPTION')[['Package']]; \
-  workflow_pkgs <- grep('^pacta[.]', workflow_pkgs, value = TRUE, invert = TRUE); \
-  pak::pak(c(gh_pkgs, workflow_pkgs)); \
+  print(gh_pkgs); \
```

Comment:
is this necessary?

Reply:
not strictly, but I've found it useful for diagnosing the string that `pak` is trying to install (in the cases where the install fails)